src/launch/launch.scss

```diff
@@ -1,3 +1,110 @@
 @import "styles/colors.scss";
+@import "styles/styles.scss";
 .page.launch {
+  display: grid;
+  justify-content: center;
+  grid-gap: 30px;
+  .banner {
+    .banner1 {
+      // Make like a heading
+      @extend .heading;
+      font-size: 4vw;
+      display: grid;
+      justify-content: center;
+    }
+    // TODO: Customize for mobile
+    .banner2 {
+      @extend .subtext;
+      // TODO: Get font from figma
+      font-family: "Lucida Sans", "Lucida Sans Regular", "Lucida Grande",
+        "Lucida Sans Unicode", Geneva, Verdana, sans-serif;
+      display: grid;
+      justify-content: center;
+      text-align: center;
+    }
+  }
+  // Let's define layouts for the launch divs
+  .launch-divs {
+    display: flex;
+    flex-flow: row wrap;
+    flex: 1;
+    width: 100vw;
```

Comment:
i'm skeptical about the 100vw here

Reply:
Yes, the grid layout uses this width min-content that just packs things together. Check it out

Reply:
vw is usually defined to include the scrollbar.  is that not the case here?